test/bin/scenario.sh

```diff
@@ -1069,6 +1080,78 @@ EOF
     fi
 }
 
+# Implementation of Gingko tests
+run_gingko_tests() {
+    local vmname="${1}"
+    shift
+
+    # Handle RUN_HOST_OVERRIDE
+    vmname=$(apply_host_override "${vmname}")
+
+    # Save current directory
+    pushd . &>/dev/null
+
+    # Check/install oc
+    "${ROOTDIR}/scripts/fetch_tools.sh" "oc" || {
+        record_junit "${vmname}" "oc_installed" "FAILED"
+        exit 1
+    }
+
+    # Check/get openshift-tests-binary
+    if ! "${ROOTDIR}/scripts/fetch_tools.sh" "ginkgo"; then
+        record_junit "${vmname}" "build_test_binary" "FAILED"
+        exit 1
+    fi
+    record_junit "${vmname}" "build_test_binary" "OK"
+
+    # Set up test environment variables
+    local -r KUBECONFIG="${SCENARIO_INFO_DIR}/${SCENARIO}/kubeconfig"
+    local -r test_results_dir="${SCENARIO_INFO_DIR}/${SCENARIO}/gingko-results"
+    mkdir -p "${test_results_dir}"
+
+    # Set up kubeconfig for tests
+    local -r vm_ip=$(get_vm_property "${vmname}" "ip")
+    local -r ssh_port=$(get_vm_property "${vmname}" "ssh_port")
+    local -r full_vmname="$(full_vm_name "${vmname}")"
+    local -r vm_hostname="${full_vmname/./-}"
+
+    # Try to get kubeconfig from multiple locations (following Robot Framework pattern)
+    if scp -P "${ssh_port}" "redhat@${vm_ip}:/var/lib/microshift/resources/kubeadmin/${vm_ip}/kubeconfig" "${KUBECONFIG}" 2>/dev/null; then
+        echo "Successfully copied kubeconfig from /var/lib/microshift/resources/kubeadmin/${vm_ip}/kubeconfig"
+    elif scp -P "${ssh_port}" "redhat@${vm_ip}:/home/redhat/kubeconfig-${full_vmname}" "${KUBECONFIG}" 2>/dev/null; then
+        echo "Successfully copied kubeconfig from /home/redhat/kubeconfig-${full_vmname}"
+    elif scp -P "${ssh_port}" "redhat@${vm_ip}:/var/lib/microshift/resources/kubeadmin/kubeconfig" "${KUBECONFIG}" 2>/dev/null; then
```

Comment:
These commands are not going to work because the access to /var/lib/microshift is closed to non-root users.